### PR TITLE
fix: correct scripts for win2k22 test lane

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
fix: correct scripts for win2k22 test lane
this commit updates names of win2k22 test files and fixes the name of efi pipeline in test script

**Release note**:
```
NONE
```

### CODE REVIEW

File: scripts/test-files/windows2k22-installer-pipelinerun.yaml [hunk 1]
```diff
@@ -1,13 +1,19 @@
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  generateName: windows2k22-installer-run-
+  generateName: windows2k22-installer-
```
Comment (0xFelix) [inline scripts/test-files/windows2k22-installer-pipelinerun.yaml:4]: Keep `-run` suffix like in other pipelineruns?

File: scripts/test-files/windows2k22-installer-pipelinerun.yaml [hunk 2]
```diff
@@ -1,13 +1,19 @@
...
   labels:
-    pipelinerun: windows2k22--installer-run
+    pipelinerun: windows2k22-installer-run
 spec:
   params:
   - name: winImageDownloadURL
     value: http://http-server/disk.img
+  - name: autounattendConfigMapName
+    value: windows2022-autounattend
```
Comment (0xFelix) [inline scripts/test-files/windows2k22-installer-pipelinerun.yaml:12]: Might rname that config map too to `2k22`?

File: scripts/test-files/windows2k22-installer-pipelinerun.yaml (hunk 1 above)
Comment (ksimon1) [reply scripts/test-files/windows2k22-installer-pipelinerun.yaml]: fixed

File: scripts/test-files/windows2k22-installer-pipelinerun.yaml (hunk 2 above)
Comment (ksimon1) [reply scripts/test-files/windows2k22-installer-pipelinerun.yaml]: fixed

File: scripts/test-files/windows2k22-installer-pipelinerun.yaml [hunk 3]
```diff
@@ -1,13 +1,19 @@
 apiVersion: tekton.dev/v1beta1
 kind: PipelineRun
 metadata:
-  generateName: windows2k22-installer-run-
+  generateName: windows2k22-installer-run
```
Comment (0xFelix) [inline scripts/test-files/windows2k22-installer-pipelinerun.yaml:4]: Keep the dash at the end?

**Pull request merged**